src/platform/backends/libvirt/libvirt_virtual_machine_factory.cpp

```diff
@@ -18,7 +18,7 @@
 #include "libvirt_virtual_machine_factory.h"
 #include "libvirt_virtual_machine.h"
 
-#include <multipass/backend_utils.h>
+#include "backend_utils.h"
```

Comment:
Why quoted and not bracketed?

There's more instances of this below. So far we've only been using quoted includes for headers in the same directory as the implementation?

Reply:
My bad, at one stage they were in same dir. Will fix

Reply:
Done, but now suspect explicitly having <shared/linux/backend_utils.h" might be clearer...

Reply:
Yeah +1 on that.

Reply:
Done